drivers/docker/driver_test.go

```diff
@@ -1455,6 +1463,12 @@ func TestDockerDriver_VolumesEnabled(t *testing.T) {
 	}
 	testutil.DockerCompatible(t)
 
+	if runtime.GOOS == "windows" {
+		// Nomad assumes : as the delimiter between host:path container, but Windows uses it for
+		// drive paths (e.g. `C:\Users...`).  Lookup volume syntax for windows and update test
+		t.Skip("TODO: Windows volume sharing doesn't work")
```

Comment:
Here, we have few bugs probably.  Our test for absolute path assumes Linux, so joins paths unexpectedly, and Docker may expect a special format here.

The failing message here is the following:
```
    driver_test.go:1469: 
        	Error Trace:	driver_test.go:1469
        	Error:      	Received unexpected error:
        	            	rpc error: code = FailedPrecondition desc = failed to create container: API error (500): invalid volume specification: 'C:\Users\appveyor\AppData\Local\Temp\1\nomad_driver_harness-264169281\ls\C:\Users\appveyor\AppData\Local\Temp\1\nomadtest_docker_volumesenabled376716010:C:\data'
```

Reply:
Docker on windows supports specifiying things like this: `-v E:\dir:C:\data` - Our code needs to be more robust to handle it.

Reply:
Makes sense - but I'll tackle it in another PR, and it probably requires some work to test against Windows vs Linux containers running on Windows.

Reply:
Makes sense - but I'll tackle it in another PR, and it probably requires some work to test against Windows vs Linux containers running on Windows.

Reply:
Makes sense - but I'll tackle it in another PR, and it probably requires some work to test against Windows vs Linux containers running on Windows.